content/md/en/docs/reference/command-line-tools/srtool.md

```diff
@@ -1,9 +1,409 @@
 ---
 title: srtool
-description: Command-line reference information for the srtool program.
+description: Describes the Docker container and command-line interface for building the Substrate WebAssembly runtime.
 keywords:
 ---
 
-The `srtool` allows building WASM runtimes in a deterministic way, allowing CIs and users to produce a strictly identical WASM runtime.
+The Substrate runtime tool (`srtool`) is a Docker container that enables you to build the Substrate WebAssembly runtime in a deterministic way.
+By using this tool, you can ensure that the same source code consistently reproduces an identical WebAssembly blob. 
+You can also use the tool to inspect and audit the runtime for any Substrate-based chain and to integrate building the WebAssembly runtime into your CI/CD pipeline.
 
-- [Go to documentation](https://github.com/paritytech/srtool).
+## Get the Docker image
+
+You must have a Docker account and Docker installed in your build environment to access the Substrate runtime tool container.
+If you sign in to the Docker Hub, you can search for the `paritytech/srtool` container and corresponding tagged images that are available there.
```

Comment:
Not sure we should start with the Docker image. It is rather easy to make mistakes.
The `srtool-cli` has been made to make things MUCH easier. I would keep this doc, but slide the docker details down for the people who want to go deeper.

Reply:
Yeah, good point. I documented using Docker first because this is literally what I did first -- before finding that the CLI makes this easier than going through Docker. Live and learn ....